src/coreclr/nativeaot/Runtime/rheventtrace.cpp

```diff
@@ -18,7 +18,6 @@
 #include "shash.h"
 #include "eventtracepriv.h"
 #include "shash.inl"
-#include "palredhawk.h"
```

Comment:
@MichalStrehovsky, if `palredhawk.h` doesn't exist in the repo or generated by cmake or a script, how does `rheventtrace.cpp` compile on windows?

Reply:
https://github.com/dotnet/runtime/blob/main/src/coreclr/nativeaot/Runtime/PalRedhawk.h

Reply:
i see. different casing. i should've used `find -iname`  :)

it built without this include